renovate.json

```diff
@@ -99,7 +99,7 @@
     },
     {
       "matchPackagePatterns": ["argoprojlabs/argocd-extension-installer"],
-      "commitMessagePrefix": "chore({{{replace 'argoprojlabs/' '' depName}}}):",
+      "commitMessagePrefix": "chore({{{replace '*' 'argo-cd' depName}}}):",
```

Comment:
hmmm ugly but it works for now, unless other argoproj (argo-workflows, agro-events, etc.) starts using the same image tags ....? 🤔

Reply:
Did you go through this list here?
https://docs.renovatebot.com/templates/

I'd start with either `parentDir` or somerhing based on `packageFile`.

Reply:
Thank you for your review. Oh there are many convenient ones. 😮 I will fix. 🙋

Reply:
I added a commit for this. 🙏 
https://github.com/argoproj/argo-helm/pull/3106/commits/7f74bba46dc2c5c8e95b5e8882d7961bac62fe00